hack/istio/install-topology-generator-demo.sh

```diff
@@ -55,7 +57,9 @@ install_topology_generator_demo() {
     $CLIENT_EXE adm policy add-scc-to-user anyuid -z default -n ${TOPO}
   fi
 
-  ${CLIENT_EXE} label namespace  ${TOPO} istio-injection=enabled --overwrite=true
+  if [ "${AUTO_INJECTION}" != "" ]; then
```

Comment:
Shouldn't this be explicitly checking for true - first, to be consistent with all the other ones you added in the other scripts, and second, because this will inject sidecars even if someone sets the value to `"false"`:
```suggestion
  if [ "${AUTO_INJECTION}" == "true" ]; then
```